receiver/awscontainerinsightreceiver/internal/stores/utils.go

```diff
@@ -101,6 +101,11 @@ func TagMetricSource(metric CIMetric) {
 		return
 	}
 
+	// Source for metric from Windows are defined in receiver
```

Comment:
nit - I don't love comments that tell me what the code is doing, they are hard to maintain.  The code should tell me what it is doing

Is there another way to structure this?

Maybe make it more generic like 

if sources are not populated, then do this.  Then the logic works for anything windows or otherwise.  A helper function might make the code more clear

The alternative is to go super explicit and make a helper

if (windows(ci)) {
   return
}

Reply:
Yeah we had a discussion and the plan is to have two functions one for linux and one for windows in utils

Reply:
Addressed this comment.
Created 2 func's 1 for linux and other for Windows. Caller is still same. This is pretty explicit now.